Docker/ods-api-web-api/alpine/mssql/run.sh

```diff
@@ -11,6 +11,12 @@ if [[ "$TPDM_ENABLED" != true ]]; then
     export Plugin__Folder="./Plugin_Disabled"
 fi
 
+if [[ "$ENCRYPT_CONNECTION" == true ]]; then
```

Comment:
$SQLSERVER_USER" &&  "$SQLSERVER_PASSWORD  required Validation is missing here which will be handled in new ticket